### PR TITLE
Changes to package structure. Implemented manage lookup with tab navigation.

### PR DESCRIPTION
**Pull request by hai2kk**
Changes to package structure. Implemented manage tab with tab navigation for stock and crypto currency menu for lookup

### CODE REVIEW

**Review by muralivp9 — APPROVED** [review]
Awesome

File: src/Manage/comonent/ManageLookup.js [hunk 1]
```diff
@@ -1,10 +0,0 @@
-import React, { Component } from "react";
-import { Text, View } from "react-native";
-
-class ManageLookup extends React.Component {
-  render() {
```
Comment (pramodsankarl) [inline src/Manage/comonent/ManageLookup.js:5]: Try avoid using `export default` instead prefer exporting then right away. `export class xxx` and prefer import it via destructuring... `import {XX} from '..'`. That way the name of the class is maintained consistently at the consumer end and easy for lookup and refactoring...

File: src/components/manage/stock/StockLookup.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
```
Comment (pramodsankarl) [inline src/components/manage/stock/StockLookup.js:1]: Why import both `React` and `Component`

File: src/navigation/ParentNavigator.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import { StackNavigator } from "react-navigation";
 import TabbedNavigationComponent from "./TabbedNavigationComponent";
-import ManageLookup from "../Manage/comonent/ManageLookup";
+import ManageLookup from "../components/manage/ManageLookup";
```
Comment (pramodsankarl) [inline src/navigation/ParentNavigator.js:4]: Good move!

File: src/components/manage/ManageLookup.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { Text, View } from "react-native";
+import Constants from "../../constants/PortfolioConstants";
+import LookupNavigator from "../../navigation/LookupNavigator"
+
+class ManageLookup extends React.Component {
+  static navigationOptions = {
+    title: Constants.MANAGE_LOOKUP_TITLE,
+    headerTintColor: Constants.FORE_COLOR_CODE,
+    headerTitleStyle: {
+      fontWeight: "300",
```
Comment (pramodsankarl) [inline src/components/manage/ManageLookup.js:11]: We should move font-family, font-size and font-weight etc to constants like font-base, font-large, font-x-large etc. That way we get consistency across app and easy to upgrade ability...

**Review by pramodsankarl — COMMENTED** [review]
Good work @hai2kk

**Pull request merged**